telegram/ext/handler.py

```diff
@@ -63,12 +81,28 @@ def __init__(self,
                  pass_update_queue=False,
                  pass_job_queue=False,
                  pass_user_data=False,
-                 pass_chat_data=False):
+                 pass_chat_data=False,
+                 use_context=None):
         self.callback = callback
         self.pass_update_queue = pass_update_queue
         self.pass_job_queue = pass_job_queue
         self.pass_user_data = pass_user_data
         self.pass_chat_data = pass_chat_data
+        self.use_context = use_context
+
+        if use_context is None:
+            if has_follow_wrapped:
```

Comment:
Since you've favoured using globals here (in contrary to code locality) please add a comment signifying which python version each of the `if... elif... else...` blocks supports.